container/public/LuigiContainer.svelte.d.ts

```diff
@@ -19,6 +19,16 @@ export default class LuigiContainer extends HTMLElement {
    */
   webcomponent: boolean;
 
+  /**
+   * If set to `true` a microfrontend will be immediately initialized. LuigiContainer sends an event `initialized` to the microfrontend.
+   */
+  can_mfe_initialized: boolean;
+
+  /**
+   * Will be set to `true` if `can_mfe_initialized` is set to `true` otherwise it will be `true` if the mfe webcomponent is attached to the container.
```

Comment:
Could you also add that this WC related. I think it doesn't apply to iframe based container microfrontends, right ?. Something like: _To be used for webcomponent based microfrontends_.